src/ScreenshotWindow.vala

```diff
@@ -70,26 +70,38 @@ public class Screenshot.ScreenshotWindow : Hdy.ApplicationWindow {
 
         backend = new ScreenshotBackend ();
 
-        all_image = new Gtk.Image.from_icon_name ("grab-screen-symbolic", DND);
+        all_image = new Gtk.Image.from_icon_name ("grab-screen-symbolic") {
+            icon_size = LARGE
+        };
 
-        var all = new Gtk.RadioButton (null) {
+        var all = new Gtk.CheckButton () {
+            active = true,
```

Comment:
Does this mean the option will always default to grabbing screen instead of the last used mode?

Reply:
That's actually now handled by the portal so this whole class goes away after merging master haha